src/lib/marketplaces/marketplaces.ts

```diff
@@ -3,15 +3,17 @@ import magicEden from "./magicEden";
 import digitalEyes from "./digitalEyes";
 import solanart from "./solanart";
 import alphaArt from "./alphaArt";
+import solSea from "./solsea";
 
 /**
  * These are the list of marketplaces that we check for notifications
  */
 const marketplaces: Marketplace[] = [
-  magicEden,
+  // magicEden,
```

Comment:
Did you mean to comment MagicEden out?

Reply:
I disabled it on my fork because it shows the txs twice with solSea. I will restore it for the main branch.